index.html

```diff
@@ -3063,6 +3076,36 @@ <h2>Subject Classification</h2>
       </p>
 
     </section>
+
+    <section>
+      <h2>Service Privacy</h2>
+      <p>
+The ability for a [=controller=] to optionally express at least one [=service=] in the [=controller document=] increases their control and agency.
+Each additional endpoint in the [=controller document=] adds privacy risk either
```

Comment:
```suggestion
Each additional endpoint in the [=controller document=] adds a privacy risk either
```

Reply:
Indeterminate privacy risk is added with each additional endpoint. It's not "one risk per endpoint". This `a` should not be inserted.